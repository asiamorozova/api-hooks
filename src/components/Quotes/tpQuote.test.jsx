import React from 'react';
import { shallow } from 'enzyme';
import Quote from './tpQuote';

describe('Quote component', () => {
  it('renders Quote', () => {
    const wrapper = shallow(<Quote 
      quoteTextOnly='some quote'
      persons='log lady'
      profanity='true'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
