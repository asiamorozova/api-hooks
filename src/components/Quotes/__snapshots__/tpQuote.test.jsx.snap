// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders Quote 1`] = `
<section>
  <p>
    some quote
  </p>
  <p>
    Quote by:
    log lady
  </p>
  <p>
    Profanity:
    true
  </p>
</section>
`;
